source/plugins/vxlan.rst

```diff
@@ -17,68 +17,47 @@
 The VXLAN Plugin
 ================
 
-System Requirements for VXLAN
------------------------------
+General
+-------
+CloudStack supports VXLAN technology to enhance scalability and flexibility in networking designs.
 
-In CloudStack 4.X.0, this plugin only supports the KVM hypervisor with the
-standard linux bridge.
+Using VXLAN (Virtual Extensible LAN) over traditional VLAN (Virtual LAN) offers several key benefits, especially for modern data centers and cloud networking environments that require high scalability and flexibility.
```

Comment:
```suggestion
Using VXLAN (Virtual Extensible LAN) instead of traditional VLAN (Virtual LAN) for layer 2 isolation method offers several key benefits, especially for modern data centers and cloud networking environments that require high scalability and flexibility.
```